src/test/com/aca/homework/week9/invoice/service/InvoiceServiceTest.java

```diff
@@ -0,0 +1,65 @@
+package com.aca.homework.week9.invoice.service;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.FileNotFoundException;
+
+class InvoiceServiceTest {
+    private InvoiceService testSubject;
+
+    @BeforeEach
+    public void setUp() throws FileNotFoundException {
+        testSubject = new InvoiceService();
+        testSubject.load("invoices.txt");
+    }
+
+    @Test
+    public void testTotalAmountByTypeSAD() {
+        Assertions.assertEquals(2602704070L, testSubject.totalAmountByType(InvoiceType.SAD));
+    }
+
+    @Test
+    public void testGetBySADType() {
+        Invoice[] filteredBySADType = testSubject.getByType(InvoiceType.SAD);
+        boolean allTypesAreSAD = true;
+        for (Invoice invoice : filteredBySADType) {
```

Comment:
please do not write a duplicate code in your test. just invoke the getByType method and assert that the result is exactly what you expect